README.md

```diff
@@ -34,6 +34,20 @@ The `setup` action automatically:
  - Set `http_proxy` and `https_proxy` environment variables to route traffic through Dev Proxy.
  - Registers a post-step that will stop Dev Proxy and clean up when the workflow completes.
 
+**Caching:**
+
+By default, the `setup` action caches the Dev Proxy installation to improve performance on subsequent workflow runs. The cache is keyed by the Dev Proxy version and runner OS. If you specify a version, that exact version will be cached. If you don't specify a version (using latest), the latest version available at the time of installation will be cached.
+
+The action automatically validates cached installations to ensure they're still functional. If validation fails, it will reinstall Dev Proxy. You can disable caching by setting `enable-cache: false`.
+
+Example with caching disabled:
+```yaml
+- name: Setup Dev Proxy
+  uses: dev-proxy-tools/actions/setup@v1
+  with:
+    enable-cache: false
+```
+
```

Comment:
We don't need this section, but we should include it as a bullet point int he list above.